docs/developer/extending/apps.mdx

```diff
@@ -102,26 +424,142 @@ mutation {
-| `CUSTOMER_CREATED`          | A new customer account is created.                                                                                               |
-| `ORDER_CREATED`             | A new order is placed.                                                                                                           |
-| `ORDER_FULLY_PAID`          | Payment is made and an order is fully paid.                                                                                      |
-| `ORDER_UPDATED`             | An order is updated; triggered for all changes related to an order; covers all other order webhooks, except for `ORDER_CREATED`. |
-| `ORDER_CANCELLED`           | An order is cancelled.                                                                                                           |
-| `ORDER_FULFILLED`           | An order is fulfilled.                                                                                                           |
-| `FULFILLMENT_CREATED`       | A new fulfillment is created.                                                                                                    |
-| `PRODUCT_CREATED`           | A new product is created.                                                                                                        |
+```graphql
+{
+  apps(first: 5){
+    edges{
+      node{
+        id
+        permissions{
+          code
+        }
+        name
+        isActive
+        webhooks{
+          name
+        }
+        
+      }
+    }
+  }
+}
+```
+
+
+The query returns simillar response:
+```json
+{
+  "data": {
+    "apps": {
+      "edges": [
+        {
+          "node": {
+            "id": "QXBwOjE=",
+            "permissions": [
+              {
+                "code": "MANAGE_ORDERS"
+              }
+            ],
+            "name": "Order App",
+            "isActive": true,
+            "webhooks": []
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+
+### Activating an App
+To activate an app, use the `appActivate` mutation. This mutation is only available to staff users with the `MANAGE_APPS` permission. 
+Staff user can't activate app which has more permissions than he has for his own. 
+
+The mutation takes the following fields:
+- `id`: the id of the `App`, required.
+
+```graphql
+mutation {
+  appActivate(id:"QXBwOjE="){
+    app{
+      id
+      isActive
+    }
+    appErrors{
+      field
+      message
+      code
+    }
+  }
+}
+```
+
+The query returns simillar response:
+```json
+{
+  "data": {
+    "appActivate": {
+      "app": {
+        "id": "QXBwOjE=",
+        "isActive": true
+      },
+      "appErrors": []
+    }
+  }
+}
+```
+### Deactivating an App
+
+To deactivate an app, use the `appDeactivate` mutation. This mutation is only available to staff users with the `MANAGE_APPS` permission. 
+Staff user can't appDeactivate app which has more permissions than he has for his own. 
```

Comment:
```suggestion
Staff user can't deactivate an app which has more permissions than he has for his own. 
```

Reply:
Also, to not assume gender ("he") you can say "Staff users [...] they have for their own" :)

Reply:
Yes, always use the neutral "they"/"their" instead of "he"/"him"/"his".